entrypoint/upgradecluster/upgradecluster_suite_test.go

```diff
@@ -17,19 +16,16 @@ import (
 var cfg *config.Product
 
 func TestMain(m *testing.M) {
-	var err error
-	flag.Var(&customflag.ServiceFlag.InstallMode, "installVersionOrCommit", "Upgrade with version or commit")
-	flag.Var(&customflag.ServiceFlag.Channel, "channel", "channel to use on install or upgrade")
-	flag.Var(&customflag.ServiceFlag.ClusterConfig.Destroy, "destroy", "Destroy cluster after test")
-	flag.Var(&customflag.ServiceFlag.SUCUpgradeVersion, "sucUpgradeVersion", "Version for upgrading using SUC")
-
-	flag.Parse()
+	customflag.AddFlags("installVersionOrCommit", "channel", "destroy", "sucUpgradeVersion")
 
+	var err error
 	cfg, err = shared.EnvConfig()
 	if err != nil {
 		return
 	}
 
+	customflag.ValidateFlags()
```

Comment:
I dont see this func being used in any other suites, is this not needed in other suites?

Reply:
good point i maybe should split  better the validation
will work on that